decayAmplitude/massDependence.cc

```diff
@@ -92,6 +92,107 @@ binnedMassDependence::amp(const isobarDecayVertex& v)
 
 ////////////////////////////////////////////////////////////////////////////////
 complex<double>
+sawtoothMassDependence::amp(const isobarDecayVertex& v)
+{
+	complex<double> amp = 0.;
+
+	const particlePtr& parent = v.parent();
+
+	const double M = parent->lzVec().M();
+
+	if (_mMin <= M && M < _mMax) {
```

Comment:
`&&` -> `and` please (you could also change it in the `binnedMassDependence` while you are at it).